asminc/pce.inc

```diff
@@ -0,0 +1,99 @@
+;
+; PCE definitions. By Groepaz/Hitmem.
+;
+
+; FIXME: optimize zeropage usage
+CURS_X          = $30
+CURS_Y          = $31
+SCREEN_PTR      = $32           ;2
+CHARCOLOR       = $34
+RVS             = $35
+BGCOLOR         = $36
+tickcount       = $37           ;4
+
+; FIXME: screen dimensions my change according to selected video mode
+screenrows      = (224/8)
+charsperline    = 61
+
+CH_HLINE        = 1
+CH_VLINE        = 2
+
+; huc6270 - Video Display Controller (vdc)
+
+VDC_MAWR        = 0             ; Memory Address Write Register
+VDC_MARR        = 1             ; Memory Address Read Register
+VDC_VWR         = 2             ; VRAM Write Register
+VDC_VRR         = 3             ; VRAM Read Register
+VDC_CR          = 4             ; Control Register
+VDC_RCR         = 5             ; Raster Counter Register
+VDC_BXR         = 6             ; Background X-Scroll Register
+VDC_BYR         = 7             ; Background Y-Scroll Register
+VDC_MWR         = 8             ; Memory-access Width Register
+VDC_HSR         = 9             ; Horizontal Sync Register (?)
+VDC_HDR         = 10            ; Horizontal Display Register (?)
+VDC_VPR         = 11            ; (unknown)
+VDC_VDW         = 12            ; (unknown use)
+VDC_VCR         = 13            ; (unknown use)
+VDC_DCR         = 14            ; (DMA) Control Register
+VDC_SOUR        = 15            ; (DMA) Source Register
+VDC_DESR        = 16            ; (DMA) Destination Register
+VDC_LENR        = 17            ; (DMA) Length Register
+VDC_SATB        = 18            ; Sprite Attribute Table
```

Comment:
Those VDC register assignments are wrong.  VWR and VRR are the same register.  And, registers 3 and 4 are not used.

After you fix that list, you must change `VDC_RCR` to `VDC_CR`, in "conio.s".